helper/tests/testing.go

```diff
@@ -0,0 +1,17 @@
+package tests
+
+import (
+	"crypto/ecdsa"
+	"github.com/0xPolygon/minimal/crypto"
+	"github.com/0xPolygon/minimal/types"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func GenerateKeyAndAddr(t *testing.T) (*ecdsa.PrivateKey, types.Address) {
+	t.Helper()
+	key, err := crypto.GenerateKey()
+	assert.NoError(t, err)
+	addr := crypto.PubKeyToAddress(&key.PublicKey)
+	return key, addr
+}
```

Comment:
The helper package is so underrated for resolving merge conflicts that arise when you add the most basic shared functionality between 2 modules 😎